docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.4"
+services:
+  cloudmapper:
+    build:
+      context: .
+    environment:
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_SESSION_TOKEN
+      - AWS_SECURITY_TOKEN
+      - AWS_DEFAULT_REGION=eu-west-1
```

Comment:
why override `AWS_DEFAULT_REGION`? i use `aws-vault`, which defaultly sets the region for me and was wondering if that could be respected in `docker-compose`

Reply:
I had issues with `AWS_DEFAULT_REGION`- not being set. It doesn't seem to matter, as the tool pulls a list of valid regions, but in order to pull a list of regions, a default region needs to be set.